src/lib/db.js

```diff
@@ -3,137 +3,55 @@ import _ from 'lodash';
 import { canonicalizeUrl } from './url.js';
 import isReserved from 'github-reserved-names';
 import Promise from 'bluebird';
+import { Activity, Creator, Donation, Thank, registerModel } from './models.js';
 
-let _db = undefined;
-
-// In the future, use private fields instead:
-//   https://github.com/tc39/proposal-class-fields
-// I thought I could use symbols, but that didn't work out.
-let modelAttrs = {};
-
-class Model {
-  constructor() {}
-
-  async save() {
-    // Does an update if the row already exists, otherwise does a put.
-    let keyname = modelAttrs[this.constructor].key;
-    let key = this[keyname];
-    let table = modelAttrs[this.constructor].table;
-    return table.add(this).catch(() => table.update(key, this));
-  }
+export { Activity, Creator, Donation, Thank };
```

Comment:
Nobody uses `Donation` and `Thank`. Maybe hide these atm to encourage use of helper functions?